### DISCUSSION TITLE
Display video/image size on presentation screen?

### DISCUSSION THREAD

**Discussion by woj-tek** [deckerst/aves#1128] [General]
Hi,
is it possible to display video/image size on image presentation? I have the details enabled so in the top section I see filename/ date|time / location and resolution but having size there would be very helpful.
It's displayed in the list view but it's less convenient

**Comment by deckerst** [comment]
Hi. It's been requested recently in #1066, but I've declined implementing it.

**Reply by woj-tek** [reply]
Hmm... that case was slightly different.

Would you be open to following suggestion(s): option/toggle between dimension and size (I don't care that much about the former but do care about the latter) or alternatively option to choose what to show in overly (filename/number of files/date/dimension/size)? :)

**Reply by deckerst** [reply]
I like the toggle idea, but I need to experiment it. Currently, when you tap on the overlay it's already handled to show the full name / description (instead of clipping it on one line), and I wonder whether it would be confusing to have a different behaviour on the subset area of the dimension/size. 

As for full customisation of the displayed fields, it's a bit tricky as the layout is dynamic, depends on available space, orientation, etc. But I agree it would be nice.

**Reply by woj-tek** [reply]
> I like the toggle idea, but I need to experiment it. Currently, when you tap on the overlay it's already handled to show the full name / description (instead of clipping it on one line), and I wonder whether it would be confusing to have a different behaviour on the subset area of the dimension/size.

Hmm... it doesn't do that for me, but the again I don't have any description I guess.

Thou having it toggle between dimension/size seems very interesting (especially if it would be "sticky", ie. remembered) :)



> As for full customisation of the displayed fields, it's a bit tricky as the layout is dynamic, depends on available space, orientation, etc. But I agree it would be nice.

Yeah, I assumed it would be from the layout perspecive.
